packages/@vuepress/bundler-vite/src/plugin/createPlugin.ts

```diff
@@ -4,6 +4,7 @@ import type { App } from '@vuepress/core'
 import type { ViteBundlerOptions } from '../types'
 import { createVuepressPlugin } from './createVuepressPlugin'
 import { createWorkaroundPlugin } from './createWorkaroundPlugin'
+import { createConstantReplacementPlugin } from './createConstantReplacementPlugin'
```

Comment:
Import order should follow alphabet order (we should set up a eslint config for that...)